bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -942,19 +942,24 @@ void createDisplay (DeviceData data) {
 }
 
 static long create32bitDIB (Image image) {
+	return create32bitDIB (image, DPIUtil.getDeviceZoom());
```

Comment:
The method without zoom is not used anywhere anymore, right? We could remove that, couldn't we?

Reply:
Removed